dotnet/src/Experimental/Process.LocalRuntime/LocalStep.cs

```diff
@@ -312,6 +324,6 @@ protected ProcessEvent ScopedEvent(ProcessEvent localEvent)
     protected ProcessEvent ScopedEvent(KernelProcessEvent processEvent)
     {
         Verify.NotNull(processEvent, nameof(processEvent));
-        return ProcessEvent.FromKernelProcessEvent(processEvent, $"{this.Name}_{this.Id}");
+        return new ProcessEvent($"{this.Name}_{this.Id}", processEvent);
```

Comment:
why is this different than `this._eventNamespace`

Reply:
Not sure, ben made this change.  I just switched from teh static factory method to the constructor since they were both equivalent _energy-levels_.